src/lib_ccaligner/ccaligner.cpp

```diff
@@ -4,7 +4,9 @@
  * Link     : https://github.com/saurabhshri
 */
 
+#include <iostream>
 #include "ccaligner.h"
+using namespace std;
 
 CCAligner::CCAligner(Params* parameters) : logFile(parameters->logPath)
```

Comment:
Sorry, but we don't want to force any namespace. Moreover, this is generally not considered a good practise specially in cases where the required code is to be used as a library.